src/console_results.sh

```diff
@@ -7,6 +7,10 @@ function console_results::render_result() {
   if [[ "$(state::is_duplicated_test_functions_found)" == true ]]; then
     console_results::print_execution_time
     printf "%s%s%s\n" "${_COLOR_RETURN_ERROR}" "Duplicate test functions found" "${_COLOR_DEFAULT}"
+    # Print the filename which contains the duplicated functions
+    printf "File with duplicate functions: %s\n" "$(state::get_file_with_duplicated_function_names)"
+    # Print the list of duplicated functions
+    printf "Duplicate functions: %s\n" "$(state::get_duplicated_function_names)"
```

Comment:
Regardless of the logic (in this context), we could remove these comments because the code below each comment is self-explanatory.